externalComm/commClass.py

```diff
@@ -22,26 +24,26 @@ def imageDownload(self, id): pass
 class FirebaseComm(Comm):
```

Comment:
We should also look at refactoring the functions in this class into protected/public parts.